roles/hydra-stack/install/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 #
 - include: tomcat.yml
 - include: solr_5.yml
-  when: solr_version | match("^5\.[0-9]\.[0-9]")
+  when: solr_version | match("^5\.\d{1,2}\.{1,2}")
```

Comment:
Makes sense. A [5-9] is easy enough to toss in the front. Once we get to 10+ we'll need to tweak things again.